docs/scos/user/features/202108.0/promotions-discounts-feature-overview.md

```diff
@@ -125,93 +125,249 @@ There are two types of discount application:
+
+1. BUY5GET1 (100 priority): $100 - $3*1.00 or $3 = $97
+2. SPICE10 (100 priority): $97 - $30*0.10 or $3.00 = $94
+3. MEMBER5 (5000 priority): $94 - $94*0.05 or $4.70 = $89.30
+4. STORE5 (5000 priority): $89.30 - $94*0.05 or $4.70 = $84.60
+
+**Calculation as displayed in cart:**
+
+Subtotal: $100.00
+
+BUY5GET10: -$3.00
+SPICE10: -$3.00
+MEMBER5: -$4.70
+STORE5: -$4.70
+
+Grand total: $84.60
+
+#### Scenario 3: Multiple discounts, both exclusive and non-exclusive.
+
+Cart subtotal: $100
+
+| DISCOUNT NAME|  DESCRIPTION|  DISCOUNT TYPE| DISCOUNT AMOUNT| DISCOUNT PRIORITY| EXCLUSIVENESS| NOTES|
+| --- | --- | --- | --- | --- |--- |--- |
+| BUY4GET1| Buy 4 baguettes, get one free|  Percentage| 100%| 100|  No| One baguette costs $3|
+| SPICE10| 10% off spices| Percentage| 10%|  100|  No| $30 of spices|
+| MEMBER5| 5% off for members| Percentage| 5%| 5000| Yes| |
+| STORE5| 5% off storewide if you spend $50 or more|  Percentage| 5%| 9000| Yes| |
+
+**Discounts applied in priority order:**
+
+MEMBER5 (5000): $100 - $100*0.05 = $95
+
+In the presence of exclusive discounts, all non-exclusive discounts are excluded.
+Between the exclusive discounts, the discount with the higher priority is chosen.
+
+#### Scenario 4: Multiple non-exclusive, percentage discount calculation types. Executing discounts in priority order results in cart subtotal no longer satisfying conditions for one or more subsequent discounts.
+
+Cart subtotal: €100
+
+| DISCOUNT NAME|  DESCRIPTION|  DISCOUNT TYPE| DISCOUNT AMOUNT| DISCOUNT PRIORITY| EXCLUSIVENESS| NOTES|
+| --- | --- | --- | --- | --- |--- |--- |
+| 50SOCKS|  50% off Nike socks| Percentage| 50%|  100|  No| Nike socks cost €40|
+| 15PANTS|  15% off all white pants|  Percentage| 15%|100|  No| White pants cost €60|
+| SAVE20| 20% off purchases over €100|  Percentage| 20%|  300|  No| |
+| SITE5|  5% off everything in the store| Percentage| 5%| 9999| No| |
+
+**Discounts applied in priority order:**
+
+1. 50SOCKS (100 priority): €100 - €40*0.5 = €80
+2. 15PANTS (100 priority): €80 - €60*0.15 = €71
+3. SAVE20 (300 priority): No longer applies, because discounts with higher priorities have reduced the subtotal below the discount condition threshold of €100.
+4. SITE5: (9999 priority): €71 - €3.55 = €67.45
+
+#### Scenario 5: Non-exclusive, fixed amount and percentage discounts with the same priority values 
+
+Cart subtotal: €100
+
+| DISCOUNT NAME|  DESCRIPTION|  DISCOUNT TYPE| DISCOUNT AMOUNT| DISCOUNT PRIORITY| EXCLUSIVENESS|
+| --- | --- | --- | --- | --- |--- |
+| 50SOCKS|  10% off Nike socks| Percentage| 10%|  100|  No| Nike socks cost €40|
+| 15PANTS|  Save €20 on all white pants|  Fixed amount| €20|100|  No| White pants cost €60|
+
+**Discounts applied in priority order:**
+
+1. 50SOCKS (100 priority): €100 - €40*0.1 = €96
+2. 15PANTS (100 priority): €96 - €20 = €76
+
+
+**Calculation as displayed in cart:**
+
+Subtotal: €100.00
+
+50SOCKS: -€4
+15PANTS: -€20.00
+
+Grand total: €76.00
+
```

Comment:
```suggestion

```

Reply:
I have commented this piece out, so we can use later, after the bug is fixed